README.md

```diff
@@ -1,16 +1,62 @@
 # Exercises
 
-__NOTE:__ DO NOT USE THE BUILT IN METHODS WHEN ASKED TO MAKE THEM! 
+__NOTE:__ DO NOT USE THE BUILT IN METHODS WHEN ASKED TO MAKE THEM!
 
 1. Using Array.prototype add a new method `myForEach` that mimics the behaivor of the built in `forEach`.
 
-2. Use your `myForEach` to add `myMap` to the Array prototype. `myMap` should behave the same as regular `map`. 
+Array.prototype.myforEach = function(callback){
+  for (let i = 0; i < this.length; i++) {
+   callback(this[i]);
+ }    
+};
 
-3. Write a 'myFilter` that behave's the same as `filter`. 
+2. Use your `myForEach` to add `myMap` to the Array prototype. `myMap` should behave the same as regular `map`.
 
-4. Write a `myEvery`. 
+Array.prototype.myMap = function(x){
+  let NewArr= [];
+  this.myforEach(num => {
+    NewArr.push(x(num));    
+  })
+  return NewArr
+}
 
-5. Write a `myReduce`. It should use the first element in the array if none is given. 
+3. Write a 'myFilter` that behave's the same as `filter`.
+
+Array.prototype.myFilter = function(callback){
+  let filArr = [];
+  this.myforEach(el => {
+    if(callback){
+      filArr.push(callback(el));
+    }
+  });
+  return filArr;
+}
+
+
+
+4. Write a `myEvery`.
+
+Array.prototype.myEvery = function(itistrue){
+
+  this.myforEach(el => {
+    if(itistrue){
+      return true;
```

Comment:
will not return out of forEach loop. Use for loop. But don't return out on true.